include/rdma/fabric.h

```diff
@@ -297,6 +297,7 @@ struct fi_info {
 	uint64_t		caps;
 	uint64_t		mode;
 	uint32_t		addr_format;
+	uint32_t		version;
```

Comment:
fi_getinfo takes a version number as input. There's got to be some sane way of handling this that doesn't modify the data structures.  If we do need to modify fi_info, I'd would rather wait until we have more feedback on what else about the API should change and hit more changes at once.